network-testing/src/main/java/com/stripe/android/networktesting/NetworkDispatcher.kt

```diff
@@ -68,7 +68,7 @@ internal class NetworkDispatcher : Dispatcher() {
         // rest of the test will continue, even if a request was missed.
         // Killing the process will ensure the test fails for a missing request even if the
         // exception is silently ignored.
-        android.os.Process.killProcess(android.os.Process.myPid())
+//        android.os.Process.killProcess(android.os.Process.myPid())
```

Comment:
revert revert!

Reply:
Oh yeah, just used for the testing. Reverted now!